src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbBase.java

```diff
@@ -113,19 +113,25 @@
     /**
      * Special default "column" key returned to client on multitenant scans. Configurable by clients if needed.
      */
-    public static final String DEFAULT_SCAN_TENANT_KEY = "mt:context";
-    public static final String DEFAULT_SCAN_VIRTUAL_TABLE_KEY = "mt:tableName";
+    private static final String DEFAULT_SCAN_TENANT_KEY = "mt:context";
+    private static final String DEFAULT_SCAN_VIRTUAL_TABLE_KEY = "mt:tableName";
```

Comment:
It looks like in other cases defaults and defaulting behavior is in the builder. To stay consistent with that, it may make sense to do the same here.